GOALS.md

```diff
@@ -0,0 +1,266 @@
+# **Detail Goals for Diamond Bluff**
```

Comment:
typo fix. do we want an opening paragraph that states that the detailed goals were derived from discussions with those companies who submitted company specific goals?